frontend/src/pages/modelServing/customServingRuntimes/useTemplates.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { listTemplates } from '~/api';
+import { TemplateKind } from '~/k8sTypes';
+import useFetchState, { FetchState } from '~/utilities/useFetchState';
+
+const useTemplates = (namespace?: string): FetchState<TemplateKind[]> => {
+  const getTemplates = React.useCallback(
+    () =>
+      listTemplates(namespace, 'opendatahub.io/dashboard=true').catch((e) => {
```

Comment:
I think you want to look to handle `namespace` not being a thing... like if not having a namespace would get you this:
`return Promise.reject(new NotReadyYet('no namespace'))`

Reply:
yeah, you are right, thanks!